### ISSUE TITLE
Voice recording button is very hard to tap on

### ISSUE THREAD

**Issue by Maay** [actorapp/actor-platform#707]
It starts working only after several taps and holds on her. Should work more smoothly

**Comment by xzenon** [comment]
A lot of users complain that it's hard to understand how to use voice recording feature.
Would be great to have popup with some simple instructions like in WhatsApp when you touch the voice recording button once.
![whatsapp](https://cloud.githubusercontent.com/assets/2015453/14643369/d0cb4b7e-0656-11e6-8012-e7ed47968b52.jpg)

**Comment by ex3ndr** [comment]
@xzenon Thanks for your feedback, but better to create separate issue
